src/partials/10_header.html

```diff
@@ -58,6 +58,11 @@
               <i class="fas fa-list-ul"></i>
               <select name="" id="">
                 <option value="">Select a category</option>
```

Comment:
W mojej opinii można w tej opcji dodać "hidden" aby ten element zniknął po wyborze kategorii

Reply:
Nie za bardzo rozumiem  co masz na myśli ?

Reply:
Teraz po rozwinięciu listy kategorii wyświetla się w środku "Select a category" a powinny być same kategorie do wyboru, gdy ustawisz atrybut hidden na <option value="">Select a category</option> to ta opcja nie będzie widoczna podczas jej rozwinięcia.

Reply:
Aa już rozumiem. Poprawione :)